src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE = 0;
+    private static final int NAME = 1;
+    private static final int WORK_HOURS = 2;
+    private static final int RATE = 3;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate from = LocalDate.parse(dateFrom, formatter);
+        LocalDate to = LocalDate.parse(dateTo, formatter);
```

Comment:
For the future, better to use the full name
```suggestion
        LocalDate localDatefrom = LocalDate.parse(dateFrom, formatter);
        LocalDate localDateto = LocalDate.parse(dateTo, formatter);
```
OR
```suggestion
        LocalDate startDate = LocalDate.parse(dateFrom, formatter);
        LocalDate endDate = LocalDate.parse(dateTo, formatter);
```